README.md

```diff
@@ -64,14 +64,6 @@ If you are on OS X and running into issues (e.g. `globstar` isn't defined and `s
     - This installs `gsed` to your command line
 
 ### Releasing
-We have automated our release process to keep it consistent among developers. Before releasing, please make sure to update the `CHANGELOG.md`.
```

Comment:
The first sentence is still practical. Let's collapse the next "To publish" onto the first one to make a new paragraph:

```md
We have automated our release process to keep it consistent among developers. To publish a new release, run:
```